tools/eslint-config-web3-base/ts-jest.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint', 'jest'],
+	extends: ['./ts', 'plugin:jest/recommended', 'plugin:jest/style'],
+	env: {
+		'jest/globals': true,
+	},
+	rules: {
+		'jest/valid-title': ['error'],
+		'jest/no-conditional-expect': ['error'],
+		'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
+		'jest/consistent-test-it': ['error'],
+		'class-methods-use-this': ['error'],
+		'@typescript-eslint/no-magic-numbers': 'off',
+		'@typescript-eslint/unbound-method': 'off',
+		'@typescript-eslint/no-require-imports': 'off',
+		'@typescript-eslint/no-explicit-any': 'off',
```

Comment:
I think this is a good rule to keep enabled

Reply:
For tests its very redundant to declare types for all kind of custom mocked data so this makes it quick to write tests like. 

```js
expect((fetchMockObject() as any).myAttr).toEqual('223')
```

Reply:
I'd prefer if even tests were strongly typed (to aid with debugging tests), but I understand your point

Reply:
I think `@typescript-eslint/no-require-imports` could still be enabled though